File: backend/src/main/java/com/bookbook/domain/chat/dto/MessageSendRequest.java
package com.bookbook.domain.chat.dto;

import com.bookbook.domain.chat.enums.MessageType;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class MessageSendRequest {
    
    @NotBlank(message = "채팅방 ID는 필수입니다.")
    private String roomId;
    
    @NotBlank(message = "메시지 내용은 필수입니다.")
    private String content;
    
    private MessageType messageType = MessageType.TEXT;
}

File: backend/src/main/java/com/bookbook/domain/chat/dto/MessageSendRequest.kt
// TODO: MessageType enum을 Kotlin으로 변환 후 import 경로 확인 필요
package com.bookbook.domain.chat.dto

import com.bookbook.domain.chat.enums.MessageType
import jakarta.validation.constraints.NotBlank

data class MessageSendRequest(
    @field:NotBlank(message = "채팅방 ID는 필수입니다.")
    val roomId: String,

    @field:NotBlank(message = "메시지 내용은 필수입니다.")
    val content: String,

    val messageType: MessageType = MessageType.TEXT
)